build.gradle

```diff
@@ -54,7 +54,7 @@ configure(allprojects) {
 		springBootVersion = project.findProperty("springBootVersion") ?: "2.1.4.RELEASE"
```

Comment:
Should we upgrade the boot version to 2.1.8.RELEASE? Also Spring Framework version 5.1.9.RELEASE, and Reactor version Californium.SR11. That's what I've updated SC OSB to use.

Reply:
We definitely should. Sadly, dependabot was not smart enough to parse that conditional. We should take a look at the other projects too.
Thanks for noticing.